packages/rolldown/tests/fixtures/tree-shake/filter-expression/_config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineTest } from "rolldown-tests";
+import { and, code, exclude, id, not } from "rolldown/filterExpr";
```

Comment:
```suggestion
import { and, code, exclude, id, not } from "rolldown/filter-expr";
```

kebab case would be commonly used for package name.

Reply:
https://github.com/rolldown/rolldown/blob/9722d32fec5f053a8433a614505610ee1360fb4e/packages/rolldown/package.json?plain=1#L49-L52, I just refer this `exports`, maybe we should talk about the naming convention for `exports` field

Reply:
On the second thought, `rolldown/filter` would be concise.